modules/customize-certificates-add-service-serving-mutating-webhook.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-mutating-webhook_{context}"]
+= Add the service CA bundle to a MutatingWebhookConfiguration
+
+You can annotate a MutatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true` to have its `caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
```

Comment:
its `caBundle` field -> the `clientConfig.caBundle` field of each webhook

Reply:
Please add a note with the following sentiment to both mutating and validating admission webhook adocs:

Admission webhook configurations needing to specify different CA bundles for different webhooks should not set the annotation since that will result in the the service CA bundle being injected for all webhooks.